apps/ejabberd/src/ejabberd.app.src

```diff
@@ -39,7 +39,8 @@
                   fast_tls,
                   usec,
                   uuid,
-                  xmerl
+                  xmerl,
+                  worker_pool
```

Comment:
Yes, it happens to be me who wrote it :) Maybe I should consider switching it to worker_pool, that's why I was asking why it is better.